app/styles/application.module.css

```diff
@@ -77,17 +77,23 @@ noscript {
     display: grid;
     grid-template-columns: auto auto 1fr auto;
     align-items: center;
+    padding: 10px;
     color: white;
 
     a {
         color: white; text-decoration: none;
         &:hover { color: darken(white, 15%); }
     }
+
+    h1 {
+        font-size: 24px;
+        margin: 0.67em 0 0.67em 10px;
```

Comment:
Instead of doing this, I recommend using `display: flex; align-items: center;` to the link that wraps both the logo and the title (and use a single link)